hazelcast.rb

```diff
@@ -0,0 +1,18 @@
+class Hazelcast < Formula
+  desc "Tool to run Hazelcast IMDG member instances locally"
+  homepage "https://github.com/hazelcast/hazelcast-command-line"
+  url "https://github.com/hazelcast/hazelcast-command-line/releases/download/v0.2-beta/hazelcast-command-line-0.2-beta.tar.gz"
+  sha256 "04bd3768769e81d60adaf456361690668ed3e71b221a0224bb398ed9bd2eb93f"
+
+  bottle :unneeded
+
+  depends_on "coreutils"
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/hazelcast"]
+    prefix.install_metafiles
+  end
+
```

Comment:
Could you please add and adapt the `test` from the old version? it'll be needed if / when we ask the main brew repo to host our formula. It basically starts an instance, checks status, and finally shuts it down.

Reply:
The issue on tests is that now we are not printing anything specific to assert for. But I will create an GH issue for the tests, and work on them later.

Reply:
Ah I see! Fine enough then, thanks!!